homeassistant/components/livisi/config_flow.py

```diff
@@ -47,30 +50,55 @@ async def async_step_user(
         else:
             controller_info: dict[str, Any] = {}
             with suppress(ClientConnectorError):
-                controller_info = await self.aio_livisi.async_get_controller()
+                controller_info = await aio_livisi.async_get_controller()
             if controller_info:
                 return await self.create_entity(user_input, controller_info)
-            errors["base"] = "cannot_connect"
 
         return self.async_show_form(
-            step_id="user", data_schema=self.data_schema, errors=errors
+            step_id="user", data_schema=DATA_SCHEMA, errors=errors
+        )
```

Comment:
Could you fill in the schema with the user_input?
```python
        data_schema = self.add_suggested_values_to_schema(DATA_SCHEMA, user_input)
        return self.async_show_form(
            step_id="user", data_schema=data_schema, errors=errors
        )
```

Reply:
Done, thank you.